pkg/webui/components/header/index.js

```diff
@@ -69,7 +69,11 @@ const Header = function({
         {!isGuest && (
           <div className={style.right}>
             {searchable && <Input icon="search" onEnter={onSearchRequest} />}
-            <ProfileDropdown className={style.profileDropdown} userId={user.ids.user_id}>
+            <ProfileDropdown
+              className={style.profileDropdown}
+              userId={user.ids.user_id}
+              data-test="header__profile_dropdown"
```

Comment:
What is the convention for test attributes?

Reply:
Yeah, this needs some further thought. I'd suggest to use a [BEM](http://getbem.com/introduction/) inspired approach, mainly to avoid ID clashes. Let's not get too strict about it though.